### PR TITLE
Release Trino Gateway chart 1.13.0

### PR DESCRIPTION
**Pull request by mosabua**
For Trino Gateway 13

Also @willmostly

### CODE REVIEW

**Comment by nineinchnick** [comment]
Trino Gateway's Docker image 13 is not available yet: https://hub.docker.com/r/trinodb/trino-gateway/tags

**Comment by mosabua** [comment]
> Trino Gateway's Docker image 13 is not available yet: https://hub.docker.com/r/trinodb/trino-gateway/tags

I know .. this is in preparation to get things ready..

**Comment by mosabua** [comment]
I pushed a commit again so this build again since the docker container is release .. but there are failures.. what am I missing @nineinchnick and @willmostly ?

**Comment by mosabua** [comment]
Nevermind .. I think I found it.

**Comment by nineinchnick** [comment]
You need to commit the updated Readme. See the main repo readme on how to set up a precommit hook to do this automatically

**Comment by mosabua** [comment]
Finally.. it passed now.

**Review by Chaho12 — APPROVED** [review]
LGTM 👍

File: charts/gateway/Chart.yaml [hunk 1]
```diff
@@ -2,15 +2,15 @@ apiVersion: v2
 name: trino-gateway
 description: A Helm chart for Trino Gateway
 type: application
-version: "0.12.0"
-appVersion: "12"
+version: "1.13.0"
```
Comment (ebyhr) [inline charts/gateway/Chart.yaml:5]: Why do we increment a major version?

**Review by willmostly — APPROVED** [review]
This lgtm

File: charts/gateway/Chart.yaml (hunk 1 above)
Comment (nineinchnick) [reply charts/gateway/Chart.yaml:5]: Having 0 as the major version might be understood the chart is unstable, or not ready for production use, which is not accurate, and might confuse some users. We agreed with @mosabua to bump major versions for both Trino and Trino Gateway to 1. So there are no breaking changes here, we just want to mark a stable release from this point forward.

Comment (mosabua) [reply charts/gateway/Chart.yaml:5]: Our chart used to have full major versions so .. and when we changed to the chart repo here it was changed to 0.x .. however we (Trino Gateway maintainers and @nineinchnick feel that this is misleading. The chart is totally usable for production so the version should be > 1.0 .. hence we are switching 1.x with x being the Trino Gateway version now .. and if we cut chart releases in between we go minor update.. 

We also plan to upgrade the trino chart to be >1 .. and the go client ..

**Pull request merged**